db/seeds.rb

```diff
@@ -1,46 +1,227 @@
+soft_rejected_proposal.ratings.create(proposal_id: soft_rejected_proposal.id, user_id: reviewer.id, score: 3)
+soft_rejected_proposal.ratings.create(proposal_id: soft_rejected_proposal.id, user_id: track_director.id, score: 1)
+soft_rejected_proposal.ratings.create(proposal_id: soft_rejected_proposal.id, user_id: speaker_reviewer.id, score: 2)
+
+withdrawn_proposal.ratings.create(proposal_id: withdrawn_proposal.id, user_id: organizer.id, score: 5)
+withdrawn_proposal.ratings.create(proposal_id: withdrawn_proposal.id, user_id: reviewer.id, score: 5)
+withdrawn_proposal.ratings.create(proposal_id: withdrawn_proposal.id, user_id: speaker_reviewer.id, score: 4)
+
+# Comments
+organizer.comments.create(proposal_id: submitted_proposal_1.id,
+                          user_id: organizer.id,
+                          body: "This looks great - very informative. Great job!",
+                          type: "PublicComment")
+
+reviewer.comments.create(proposal_id: soft_accepted_proposal.id,
+                         user_id: reviewer.id,
+                         body: "Oh my goodness, this looks so fun!",
+                         type: "PublicComment")
+
+track_director.comments.create(proposal_id: soft_accepted_proposal.id,
+                               user_id: track_director.id,
+                               body: "Cleary we should accept this talk.",
+                               type: "InternalComment")
+
+speaker_reviewer.comments.create(proposal_id: withdrawn_proposal.id,
+                                 user_id: speaker_reviewer.id,
+                                 body: "Uhhhh... is this for real? I can't decide if this is amazing or insane.",
+                                 type: "InternalComment")
+
+
+### SapphireConf -- this is an event in the early set-up/draft stage
+sapphire_start_date = 10.months.from_now
+
+# Core Event Info
+sapphire_guidelines = %Q[
+# SapphireConf - The place to be in 2017!
+
+The first annual conference about the hot new programming language Sapphire
+is happening in moody Reykjavík, Iceland on February 1st, 2017!
+
+If you are on the cutting edge with savvy Sapphire skills, we want you!
+
+## #{Faker::Hipster.sentence(4)}
+#{Faker::Hipster.paragraph(20)}
+]
+
+sapphire_event = Event.create(name: "SapphireConf",
+                              slug: "sapphireconf",
+                              url: "https://en.wikipedia.org/wiki/Sapphire",
+                              contact_email: "info@sapphire.event",
+                              start_date: sapphire_start_date,
+                              end_date: sapphire_start_date + 1,
+                              guidelines: sapphire_guidelines,
+                              proposal_tags: %w(beginner intermediate advanced),
+                              review_tags: %w(beginner intermediate advanced))
+
+# Event Team
+sapphire_event.teammates.create(user: organizer, email: organizer.email, role: "organizer", state: Teammate::ACCEPTED)
```

Comment:
We can definitely use the Faker gem much more than I am currently, but I didn't think it was worth putting more time into this right now - there's a lot of great information that loads in with what we currently have.